helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -164,12 +164,10 @@ splunkObservability:
 #   otel-collector agent for further processing.
 # - `otel`: utilize native OpenTelemetry log collection.
 #
-# Change it to `otel` to get higher throughput performance and avoid installing
-# an extra container for fluentd.
+# `fluentd` will be deprecated soon, so it's recommended to use `otel` instead.
 ################################################################################
 
-# The default value is "fluentd" which will be changed to "otel" in 0.86.0.
-logsEngine: ""
+logsEngine: otel
```

Comment:
should we bump the default resource limits with the logs collection being done by the collector container? I didn't see any guidelines for how much more resources the collector container should be provided

Reply:
I think we should keep it as is until we find out that the current defaults are too low. Logs collection is disabled by default for o11y. For `splunkPlatform`, it always depends on the volume. From my tests, I remember it was not that significant increase after switching from fluentd.

Reply:
Ok, hopefully users are mindful of setting the limits to match their usage.